app/settings.py

```diff
@@ -9,6 +8,44 @@
 load_dotenv(dotenv_path=env_path)
 
 # FLASK SETTINGS
-SECRET_KEY: tp.Optional[str] = os.environ.get('SECRET_KEY')
-FLASK_APP: tp.Optional[str] = os.environ.get('FLASK_APP')
-FLASK_ENV: tp.Optional[str] = os.environ.get('FLASK_ENV')
+SECRET_KEY: str = os.environ.get('SECRET_KEY')  # type: ignore
+FLASK_APP: str = os.environ.get('FLASK_APP')  # type: ignore
+FLASK_ENV: str = os.environ.get('FLASK_ENV')  # type: ignore
+
+# DB SETTINGS
+POSTGRES_HOST: str = os.environ.get('POSTGRES_HOST')  # type: ignore
+POSTGRES_USER: str = os.environ.get('POSTGRES_USER')  # type: ignore
+POSTGRES_PASSWORD: str = os.environ.get('POSTGRES_PASSWORD')  # type: ignore
+POSTGRES_DB: str = os.environ.get('POSTGRES_DB')  # type: ignore
+POSTGRES_PORT: str = os.environ.get('POSTGRES_PORT')  # type: ignore
+
+# TEST DB SETTINGS
+TEST_POSTGRES_HOST: str = 'db-test'
```

Comment:
why you need test db notion? you should just set correct db credentials during tests

Reply:
this is for creating new account and etc and drop test data